CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## 0.8.0-dev
 
+### Fixed
+
+- Assertion failure on 32-bit architectures
```

Comment:
This is a bit unspecific as to what the actual issue is, I'd propose something more on the nose:

```suggestion
- Crash due to assertion failure on 32-bit architectures
```